src/code/code_8012EC80.c

```diff
@@ -545,6 +545,9 @@ void func_8012F1BC(s16 arg0);
 #pragma GLOBAL_ASM("asm/non_matchings/code/code_8012EC80/func_8012F1BC.s")
 #endif
 
+/**
+ * Returns the skullTokenCount for the corresponding scene
+ */
 s16 func_8012F22C(s16 sceneIndex) {
```

Comment:
Yeah, adding on, this will be overwritten by a different PR. So you can probably just leave this out?

Reply:
You mean remove the comment?

Reply:
I guess it doesn't really matter... It's just an extra conflict. It's named `Inventory_GetSkullTokenCount` in the other open PR, which already has everything described in the comment